Makefile

```diff
@@ -333,12 +331,19 @@ reconfigure:
 	./configure $(CONFIGURE_ARGS)
 endif
 
+utils/domainstate.ml: utils/domainstate.ml.c runtime/caml/domain_state.tbl
+	$(CPP) -I runtime/caml $< > $@
+
+utils/domainstate.mli: utils/domainstate.mli.c utils/domainstate.ml \
```

Comment:
Very minor: I don't think that this build rule should depend on utils/domainstate.ml.

Reply:
Fixed.